js/app/src/components/Number/Number.svelte

```diff
@@ -29,6 +31,8 @@
 
 	<Number
 		bind:value
+		bind:max
```

Comment:
no need to bind max and min, can just use `{max} {min}` instead, as values will only be passed down and cannot be passed back up